chainer/datasets/kmnist.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import numpy
+
+import chainer
+from chainer.dataset import download
+from chainer.datasets._mnist_helper import make_npz
+from chainer.datasets._mnist_helper import preprocess_mnist
+
+
+_kmnist_labels = [('o', u'\u304A'), ('ki', u'\u304D'),
+                  ('su', u'\u3059'), ('tsu', u'\u3064'),
+                  ('na', u'\u306A'), ('ha', u'\u306F'),
+                  ('ma', u'\u307E'), ('ya', u'\u3084'),
+                  ('re', u'\u308C'), ('wo', u'\u3092')]
+
+
+def get_kmnist_labels():
+    """Provide a list of labels for the KMNIST dataset.
```

Comment:
A suggestion, for consistency.
```suggestion
    """Provides a list of labels for the KMNIST dataset.
```